docs/manual/config/index.rst

```diff
@@ -163,7 +163,10 @@ configuration variables that control specific aspects of Qtile's behavior:
       - dict(font='sans',
              fontsize=12,
              padding=3)
-      - Default settings for bar widgets.
+      - Default settings for bar widgets.  Note: if the font file
+        associated with the font selected here is modified while Qtile
+        is running, Qtile may segfault (for details see `issue #2656
+        <https://github.com/qtile/qtile/issues/2656>`_).
```

Comment:
Can we just leave this one? Doesn't feel like we need to say the same thing three times.

Reply:
Sure :+1:  updated the patch.